authapi/models.py

```diff
@@ -9,14 +9,24 @@ class SeedOrganization(models.Model):
     modified_at = models.DateTimeField(auto_now=True)
     archived = models.BooleanField(default=False)
 
+    def get_active_teams(self):
+        return self.seedteam_set.filter(archived=False)
+
 
 class SeedPermission(models.Model):
-    # TODO: Implement SeedPermission model
-    pass
+    permission_type = models.CharField(max_length=128)
+    object_id = models.CharField(max_length=64)
+    namespace = models.CharField(max_length=64)
+    created_at = models.DateTimeField(auto_now_add=True)
+    modified_at = models.DateTimeField(auto_now=True)
```

Comment:
can I suggest you use `updated_at` to be consistent with other seed service APIs?